submissions/nwokejeme-immaculeta/dom/signup.js

```diff
@@ -0,0 +1,54 @@
+let submitForm = document.getElementById('validateForm'),
+    firstName = document.getElementById('firstName'),
+    lastName = document.getElementById('lastName'),
+    otherName = document.getElementById('otherName'),
+    phoneNumber = document.getElementById('phoneNumber'),
+    email = document.getElementById('email');
+    password = document.getElementById('password');
+    confirmPassword = document.getElementById('confirmPassword');
+
+    let validate = (field, event, regEx, result, output) => {
+        if (field.value === "") {
+            field.nextElementSibling.innerHTML = result;
+            event.preventDefault();
+            return false;
+        } else if (!field.value.match(regEx)) {
+            field.nextElementSibling.innerHTML = output;
+            event.preventDefault();
+            return false;
+        } else {
+            field.nextElementSibling.innerHTML = "";
+            return true;
+        }
+    }
+
+    function confirmPasswordVal(event){
+        if (confirmPassword.value == ''){
```

Comment:
This first check is can necessary here. Kindly use the `validate` function to check if `confirmPassword` is empty